src/packages/@ncigdc/components/Header.js

```diff
@@ -36,6 +36,7 @@ const styles = {
 };
 
 const Header = compose(
+  withRouter,
```

Comment:
This probably isn't needed if https://github.com/NCI-GDC/portal-ui/pull/1982 is merged right?